site/src/routes/api/v1/[...slug]/+server.ts

```diff
@@ -0,0 +1,21 @@
+import type { RequestEvent } from "./$types";
```

Comment:
Typescript is a bit of a pain to find the correct typing for the svelte stuff, not sure how long it'll take to get used to this.